onnx/backend/test/case/node/dequantizelinear.py

```diff
@@ -85,18 +86,62 @@ def export_e4m3fn() -> None:
             name="test_dequantizelinear_e4m3fn",
         )
 
+    @staticmethod
+    def export_e4m3fn_float16() -> None:
+        node = onnx.helper.make_node(
+            "DequantizeLinear",
+            inputs=["x", "x_scale"],
+            outputs=["y"],
+            axis=0,
+        )
+
+        # scalar zero point and scale
+        x = make_tensor("x", TensorProto.FLOAT8E4M3FN, [5], [0, 0.5, 1, 448, -104])
+        x_scale = np.float16(2)
+        y = np.array([0.0, 1.0, 2.0, 896.0, -208.0], dtype=np.float16)
```

Comment:
NIP, I wonder whether we shall use some code to generate expected output (y) or hard code it like this? Maybe comment on how the numbers are generated is sufficient in case it is necessary, for example, to create new test cases.

Reply:
The test case are both used to document and to check the operators for a specific runtime but we can create another subfolder in onnx/backend/test/data for a better coverage. The following pr i'm working on makes it easier to create test cases with ReferenceEvaluator.